docs/elements.md

```diff
@@ -1891,6 +1891,61 @@ element contents only in a specific panel.
 
 ---
 
+### `pl-hidden-hints` element
+
+Display progressive hints that become accessible as the number of student submissions increases for the current variant.
+Hints are only open on page load when they are first revealed (when first reaching the desired submission count).
+Otherwise hints start closed and must be opened by the user. The submission counter is reset when new variants are
+generated. In particular, **this element does not reveal new hints across variants.**
```

Comment:
I haven't fully thought this through, but I imagine that hints work best when there's a penalty for retrying; otherwise a student could just submit the same wrong answer a bunch of times to reveal all the hints. Is that how you use it in practice? Do you think it's worth mentioning that in the docs here?

Reply:
I think that it's worth mentioning. In our use case, the student is allowed full credit on all attempts, but there's a 1min delay between submissions. Since the hints are hidden, a student would have to sit there for a few minutes on each question to see if any were going to be revealed, and we've found this is enough of a deterrent to prevent the situation you're describing.

Reply:
Added some of the discussion from here and addressed all other comments @nwalters512.